Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM python:3.7.2-alpine3.8
+
+WORKDIR /opt/slack-export-viewer
+
+COPY requirements.txt ./
+RUN pip3 install --no-cache-dir -r requirements.txt
+
+COPY . /
+
+EXPOSE 5000
+
+RUN ["chmod", "+x", "/entrypoint.sh"]
```

Comment:
```suggestion
RUN  chmod 755 entrypoint.sh
```

Reply:
), sorry

Reply:
This shouldn't be necessary as `git` preserves file modes. If you just `chmod 755 entrypoint.sh` in the repo, we can probably discard this.

Reply:
i'll check

Reply:
done with chmod